hotkey.js

```diff
@@ -70,6 +70,8 @@ function findHotKey(event, window, transformers) {
   let code = prefix
   if (event.key === '+') {
     code += 'plus'
+  } else if (event.key === '') {
+    code += 'space'
   } else {
     code += event.key.toLowerCase()
   }
```

Comment:
This approach can save a some bytes (it requires `2130 B` rather than `2135 B`), but I'm not sure it worth that.

```suggestion
  let replacers = { '': 'space', '+': 'plus' }
  code += replacers[event.key] ?? event.key.toLowerCase()
```

Reply:
Yes, let’s go with `replacers` way (just define `const` outside of the function). It is better for readability (we may have more replacements like this).

Reply:
Have to use `let` this way:
```
let KEY_REPLACERS = {
  '': 'space',
  '+': 'plus'
}
```


Otherwise, `size-limits`

Reply:
Sure!

Reply:
Done ✅